apps/application/flow/step_node/question_node/impl/base_question_node.py

```diff
@@ -93,7 +93,7 @@ def execute(self, model_id, system, prompt, dialogue_number, history_chat_record
         question = self.generate_prompt_question(prompt)
         self.context['question'] = question.content
         system = self.workflow_manage.generate_prompt(system)
-        self.context['system'] = question.content
+        self.context['system'] = system
         message_list = self.generate_message_list(system, prompt, history_message)
         self.context['message_list'] = message_list
         if stream:
```

Comment:
There is no apparent issue with this code. However, there are a couple of small improvements that can be made for readability and maintainability:

1. **Consistent Use of `content`**: Since you're already referring to the `system.content`, it would make sense to consistently use `.content` throughout.

2. **Use of Variables For Clarity**: While current approach works, using variables can enhance clarity:
   ```python
     question = self.generate_prompt_question(prompt)
     self.context['question'] = question.content

     # Using 'self.system' to avoid redundancy
     system_content = self.workflow_manage.generate_prompt(system).content
     self.context['system'] = system_content

     message_list = self.generate_message_list(system, prompt, history_message)
     self.context.update({'message_list': message_list})
 ```

These changes improve consistency within your codebase and potentially reduce potential errors or misunderstandings down the line.